duckduckgo/iOS
# Migrate to TrackerRadarKit

Task/Issue URL: https://app.asana.com/0/72649045549333/1194161438970554/f
Tech Design URL: https://app.asana.com/0/481882893211075/1199405653869875/f
CC: @SlayterDev 

**Description**:

This PR migrates the app to the new TrackerRadarKit Swift package.

The changes in this PR are fairly simple: a handful of source files have been removed, and imports have been updated. There are still tests which depend on `TrackerRadarKit` in the iOS project, in cases where they also depend on integral parts of the iOS app. Additionally, integration tests have not been removed – am open to feedback here.

**Steps to test this PR**:
1. Run unit & integration tests to verify that there are no failure
1. Test that content blocking functionality all works as expected, including verifying that temporary unprotected domains and domains without privacy protection are excluded as expected

**TODO**:

* [x] Move to a fixed commit in the TrackerRadarKit repo once https://github.com/duckduckgo/TrackerRadarKit/pull/1 has been merged

**Device Testing**:


**OS Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
